src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.stepev.test.training.at.hw1;
```

Comment:
Please rebase current hw to the latest `master` branch.
In each PR should be only files related to the current homework. 
For the HW3 and other please create new branch from the `master` and not previous homework branch